### PR TITLE
added e2e example test

### PR DESCRIPTION
**Pull request by rluvaton**
added simplified example that test the build and installation of the package

### CODE REVIEW

**Comment by rluvaton** [comment]
Hey man @juanpicado do you know why the build is failing?

**Review by juanpicado — APPROVED** [review]
Awesome ! Thanks good to see u again !

**Comment by juanpicado** [comment]
> Hey man @juanpicado do you know why the build is failing?

Yes is my fault, I need to fix the build for deploy the website but I was in 🏝️ but :) just finished probably fix this week.

**Comment by rluvaton** [comment]
> Awesome ! Thanks good to see u again ! 

Great to see you too

**Comment by rluvaton** [comment]
Hey @juanpicado, it seems like the website did not deploy yet, is there anything I can do to help?

**Comment by juanpicado** [comment]
Mmmm, it deploys every day automatically, 🤔 all green

https://github.com/verdaccio/verdaccio/actions/workflows/website.yml

Screenshot 2023-07-03 at 13 22 59

File: website/docs/e2e.md [hunk 1]
```diff
@@ -96,3 +96,4 @@ Via CLI:
 - [e2e-ci-example-gh-actions](https://github.com/juanpicado/e2e-ci-example-gh-actions)
 - [verdaccio-end-to-end-tests](https://github.com/juanpicado/verdaccio-end-to-end-tests)
 - [verdaccio-fork](https://github.com/juanpicado/verdaccio-fork)
+- [simplified e2e testing using verdaccio](https://github.com/rluvaton/e2e-verdaccio-example)
```
Comment (juanpicado) [inline website/docs/e2e.md:99]: @rluvaton ahhh, you updated main docs, which is deployed at 6.x https://verdaccio.org/docs/next/e2e for 5.x you need to update same file in the versioned folder.

Comment (rluvaton) [reply website/docs/e2e.md:99]: Thanks, created a new pr

**Pull request merged**